en/docs/api-management/api-policies.md

```diff
@@ -0,0 +1,383 @@
+# API Policies
+
+API Policies are units of business logic that you can apply to modify the flow of API invocations. 
+
+You can apply a policy to alter the  `Request`, `Response`, or `Error` flow of an API invocation before it reaches the backend or the client. For example, you can add a policy to the response flow to transform the payload from JSON to XML and add a header to the response. 
+
+Choreo includes a set of default policies that address common use cases. You can easily attach one or more policies to an API proxy component implementation via the Choreo Console. If necessary, you can also rearrange or swap the policies you attach.
+
+In Choreo, when you attach a mediation policy to a proxy, the deployment is a two-step process.
+
+1. Deployment initiation: 
+
+     If the component to which you want to attach the mediation policy is new, the system creates and commits a new repository with the mediation service code based on the attached policy. This new service is called the interceptor application. 
+
+2. Deploying the API:
+
+     Once the deployment initiation is complete, you can specify configuration values if any, and proceed to deploy. Choreo builds the generated interceptor application and pushes the Docker image to the Docker registry. Finally, Choreo deploys the interceptor application with the mediation service and the API Proxy.
+
+When a mediation policy is attached to a specific flow, the API invocation undergoes the following behavioral modification:
+ 
+ ![Request/Response flow](../assets/img/api-management/api-policies/request-response-flow.png)
+
+ - In the request path, the requests that pass through the gateway reach the relevant component, and Choreo executes any attached policies to the resource's request path before sending it to the backend. 
```

Comment:
Reaching the relevant component is quite confusing. shall we say Choreo instead?

Reply:
Also these policies are executed in the gateway right? I am not sure if we want to describe the components related to it?

Reply:
`requests that pass through the gateway reach the interceptor app, and executes any attached policies to the resource's request path before sending it to the backend.`